src/drive/lib/migration/qualification.js

```diff
@@ -0,0 +1,266 @@
+import { models } from 'cozy-client'
+import log from 'cozy-logger'
+const { Qualification } = models.document
+const { saveFileQualification } = models.file
+import { get, has, isEmpty, omit, sortBy } from 'lodash'
+
+/**
+ * Query the files indexed on their updatedAt date.
+ *
+ * @param {object} client - The CozyClient instance
+ * @param {string} date - The starting date to query
+ * @param {number} limit - The maximum number of files to return
+ */
+export const queryFilesFromDate = async (client, date, limit) => {
+  const query = client
+    .find('io.cozy.files')
```

Comment:
client.find() is deprecated, you should use Q('io.cozy.files').where() (see https://github.com/cozy/cozy-client/pull/830)

Reply:
Just when I got used to this API, it is deprecated, my story with js :laughing:

Reply:
Done with 0290faa65425b519a88fb9d7dc290624c5410a4e